doc/sphinx/reactors.rst

```diff
@@ -195,3 +181,228 @@ temperature:
+
+ .. math:: \dot m = K_v (P_1 - P_2)
+
+ if :math:`P_1 > P_2.` Otherwise, :math:`\dot m = 0`. However, an arbitrary function can also be specified, such that
+
+ .. math:: \dot m = F(P_1 - P_2)
+
+ if :math:`P_1 > P_2`, or :math:`\dot m = 0` otherwise. It is never possible for the flow to reverse and go from the downstream to the upstream reactor/reservoir through a line containing a Valve object.
+
+ Valve objects are often used between an upstream reactor and a downstream reactor or reservoir to maintain them both at nearly the same pressure. By setting the constant :math:`K_v` to a sufficiently large value, very small pressure differences will result in flow between the reactors that counteracts the pressure difference.
+
+- **Mass Flow Controller**: A mass flow controller maintains a specified mass flow rate independent of upstream and downstream conditions. The equation used to compute the mass flow rate is
+
+ .. math:: \dot m = \max(\dot m_0, 0.0)
+
+ where :math:`\dot m_0` is either a constant value or a function of time. Note that if :math:`\dot m_0 < 0`, the mass flow rate will be set to zero, since reversal of the flow direction is not allowed.
+
+ Unlike a real mass flow controller, a MassFlowController object will maintain the flow even if the downstream pressure is greater than the upstream pressure. This allows simple implementation of loops, in which exhaust gas from a reactor is fed back into it through an inlet. But note that this capability should be used with caution, since no account is taken of the work required to do this.
+
+- **Pressure Controller**: A pressure controller is designed to be used in conjunction with another 'master' flow controller, typically a MassFlowController. The master flow controller is installed on the inlet of the reactor, and the corresponding PressureController is installed on on outlet of the reactor. The PressureController mass flow rate is equal to the master mass flow rate, plus a small correction dependent on the pressure difference:
+
+ .. math:: \dot m = \dot m_{\rm master} + K_v(P_1 - P_2).
+
+
+Time Integration
+----------------
+
+Cantera provides an ODE solver for solving the stiff equations of reacting systems. If installed in combination with SUNDIALS, their optimized solver is used. Starting off the current state of the system, it can be advanced in time by two methods:
+
+- ``step``\ `(\Delta t_{\rm max-estimate})`: The step method computes the state of the system at the a priori unspecified time `t_{\rm new}`. The time `t_{\rm new}` is internally computed so that all states of the system only change within a (specifiable) band of absolute and relative tolerances. Additionally, the time step must not be larger than a predefined maximum time step `\Delta t_{\rm max}`. `\Delta t_{\rm max-estimate}` is used to estimate `\Delta t_{\rm max}` for the very first time step and is ignored in subsequent calls. The new time `t_{\rm new}` is returned by this function.
```

Comment:
Without seeing the output HTML, its hard to read this, but I would just note that a commit was recently made to remove the argument to the `step` method, so its called as `netw.step()` now. This change was made in 8d4e9bff6e0dccd3ff8fb9589d03eb155afe5d7e